crates/sui-framework/src/natives/object.rs

```diff
@@ -14,7 +14,7 @@ use move_vm_types::{
 use smallvec::smallvec;
 use std::collections::VecDeque;
 
-pub fn bytes_to_address(
+pub fn address_from_bytes(
```

Comment:
I started writing the tests, I realised this comment (it's below, I can't highlight it because I didn't change it) is no longer true:

>     // unwrap safe because this native function is only called from new_from_bytes,
>     // which already asserts the size of bytes to be equal of account address.

(`new_from_bytes` doesn't exist anymore, and we're also exposing it publicly, so this path needs to return a `NativeResult:err`)

But also, I wanted to double check: @damirka -- do we want to have this function accept raw bytes, or BCS encoded bytes?  I'm asking because all the functions that go in the other direction use `bcs::to_bytes`, but currently this function just expects raw bytes coming in.

Reply:
No difference between these two. In bcs address is a sequence of bytes.